docs/js-api.md

```diff
@@ -8,7 +8,7 @@ This example uses the OpenAPI v3 rules to lint a document.
 
 ```js
 const { Spectral } = require('@stoplight/spectral');
-const { oas3Functions, rules: oas3Rules } = require('@stoplight/spectral/rulesets/oas3');
+const { oas3Functions, rules: oas3Rules } = require('@stoplight/spectral/dist/rulesets/oas3');
```

Comment:
@P0lip Why? Didn't we copy the rulesets to the main directory as well?

Reply:
Only the JSON files.

Reply:
Meh